tsconfig.json

```diff
@@ -1,6 +1,6 @@
 {
   "compilerOptions": {
-    "target": "esnext",
+    "target": "ES2019",
```

Comment:
older es spec needs to get targeted to avoid runtime error with optional chaining